### PR TITLE
[doc] Revise documentation prose

### PR DESCRIPTION
**Pull request by vlpannel**
This PR should follow #164 (which includes cd4593f1e404afe5d0a616f0a568e223d3cc4870 through a5808fef062342eb1d847af534d2dd119f09185c).

Prose revisions for a more coherent set of docs. This PR revises a number of files, including:
* Hardware development stages prose
* Primitive documentation prose
* DV methodology doc prose
* The DV components section of the documentation
* Earlgrey docs' prose
* Darjeeling docs
* Chip signing docs
* Code vendoring docs

Generally, documentation should be more advisory than policy-enforcing, as the Pavona project's policies should be headed by its TSC. Also, all top names are now standardized to be one word ("Earlgrey", "Englishbreakfast"). Outdated information has been removed, typos fixed.

### CODE REVIEW

File: doc/contributing/dv/methodology/README.md [hunk 1]
```diff
@@ -140,45 +123,42 @@ A few examples of these are as follows:
...
 ### IP Level DV
 
 IP level DV testbenches are small and provide fine grained control of stimulus and corner case generation.
 Tests at this level run relatively quickly and development cycles are shorter.
 Coverage closure is more intensive since there are typically no pre-verified sub-modules.
-To achieve our coverage goals, we take a constrained random approach to generate the stimulus.
+By default, this repo takes a constrained random approach to generate the stimulus.
 The DV environment models the behavior of the DUT more closely to perform checks (typically within the scoreboard or via SVAs) independently of the stimulus.
-In some IPs, specifically the ones that provide cryptographic functions, we also use open source third party C libraries as reference models to check the behavior of the DUT through DPI-C calls.
+For some IPs, specifically the ones that provide cryptographic functions, DV will integrat open-source, third-party C libraries as reference models to check the behavior of the DUT through DPI-C calls.
```
Comment (qmn) [inline doc/contributing/dv/methodology/README.md]:
```suggestion
For some IPs, specifically the ones that provide cryptographic functions, DV will integrate open-source, third-party C libraries as reference models to check the behavior of the DUT through DPI-C calls.
```

File: doc/contributing/dv/methodology/README.md
Comment (qmn) [inline doc/contributing/dv/methodology/README.md]:
```suggestion
This dashboard contains information in a tabulated format mapping the written tests to planned tests (in the testplan) to provide the ability to track progress towards executing the testplan.
```

File: doc/contributing/dv/methodology/README.md [hunk 2]
```diff
@@ -228,18 +206,16 @@ These set of tests ensure that those requirements are indeed met.
 #### Security & Error Tests
 
 Error tests focus on generating stimulus that may not be considered legal and ensure that the DUT can detect, react and behave accordingly, instead of locking up or worse, exposing sensitive information.
-These types of tests are particularly important for a security chip such as ours.
+These types of tests are particularly important for a security-centered designs.
```
Comment (qmn) [inline doc/contributing/dv/methodology/README.md]:
```suggestion
These types of tests are particularly important for security-centered designs.
```

File: doc/contributing/dv/methodology/README.md [hunk 3]
```diff
@@ -291,37 +264,32 @@ The main goals (for all DUTs) are:
...
+
+Weekly regressions are even more comprehensive than nightlies.
 
 ## Coverage Collection
 
-Collecting, analyzing, and reporting coverage with waivers is a requirement to assert 'verification complete'.
-Any gaps in our measured coverage need to be understood and either waived (no need to cover) or closed by additional testing.
-The end goal is to achieve 100% coverage across all applicable coverage metrics.
-This process is known as "coverage closure", and is done in close collaboration with the designer(s).
+Verification completeness relise on collecting, analyzing, and reporting coverage with waivers.
```
Comment (qmn) [inline doc/contributing/dv/methodology/README.md]:
```suggestion
Verification completeness relies on collecting, analyzing, and reporting coverage with waivers.
```

File: doc/contributing/dv/methodology/README.md [hunk 4]
```diff
@@ -291,37 +264,32 @@ The main goals (for all DUTs) are:
...
 **Why do we need coverage?**
 
-The common answer is to flush out bugs in the design.
-This is not accurate enough.
-Making sure there are no bugs in a design is important, but not sufficient.
-One must also make sure the design works as intended.
-That is, it must provide all the functionality specified in the design specification.
-So a more precise answer for why we need coverage is to flush out flaws in the design.
-These flaws can be either design bugs or deficiencies in the design with respect to the specification.
+The common answer is to flush out bugs in the design, but is is also important to make sure the design works as intended.
```
Comment (qmn) [inline doc/contributing/dv/methodology/README.md]:
```suggestion
The common answer is to flush out bugs in the design, but also to make sure the design works as intended.
```

File: doc/getting_started/setup_verilator.md
Comment (qmn) [inline doc/getting_started/setup_verilator.md:1]: With #62 merged, this doc may require some updates (we no longer have the requirement to use Verilator 4.210).

File: hw/formal/README.md
Comment (qmn) [inline hw/formal/README.md:1]: I believe this got changed in #164.

File: hw/ip/prim/README.md [hunk 5]
```diff
@@ -289,28 +283,25 @@ filesets:
...
- It serves as an example for partners to integrate their own technology specific prim library.
 
-The steps in [creating a tech library](#creating-a-technology-library) were followed to create this prim library.
+### `prim_asap7` example
 
+[ASAP7](https://github.com/The-OpenROAD-Project/asap7) is an open-source standard-cell library.
 Each standard-cell instance name is prefixed with a `u_size_only_` such that these instances can be easily identified during synthesis and preserved.
 
-### Important synthesis constraints to keep important redundant constructs
+#### Important synthesis constraints for keep important redundant constructs
```
Comment (qmn) [inline hw/ip/prim/README.md]:
```suggestion
#### Important synthesis constraints for keeping important redundant constructs
```

**Review by qmn — APPROVED** [review]
Couple of small typos but LGTM! Thanks! Be sure to rebase (getting rid of the duplicate commits from #164) before merging.

File: doc/getting_started/setup_verilator.md
Comment (vlpannel) [reply doc/getting_started/setup_verilator.md:1]: Should this doc simply be deleted? Particularly if people no longer need to build Verilator themselves.
